API/fleece/FLSlice.h

```diff
@@ -86,7 +86,7 @@ static inline FLSlice FLStr(const char *str) {
 // Macro version of FLStr, for use in initializing compile-time constants.
 // STR must be a C string literal. Has zero runtime overhead.
 #ifdef _MSC_VER
-    #define FLSTR(STR) {("" STR), sizeof(("" STR))-1}
+    #define FLSTR(STR) (FLSlice({("" STR), sizeof(("" STR))-1}))
```

Comment:
Are you sure this is valid C?

Reply:
It's not....ugh....but the original is not great C++ either and causes rampant errors when used inline :(